package.json

```
@@ -6,11 +6,13 @@
     "generate": "nuxt generate",
     "preview": "nuxt preview",
     "postinstall": "nuxt prepare",
-    "prepare": "husky install"
+    "prepare": "husky install",
+    "lint": "npx prettier --write . & git add ."
```

```suggestion
    "lint": "lint-staged"
```

需要下载 `lint-staged` 依赖